apps/molgenis-components/src/components/forms/EditModal.vue

```diff
@@ -285,10 +292,12 @@ export default {
     this.schemaMetaData = await this.client.fetchSchemaMetaData();
     const settings: ISetting[] = await this.client.fetchSettings();
 
-    this.useChapters =
-      settings.find(
-        (item: ISetting) => item.key === IS_CHAPTERS_ENABLED_FIELD_NAME
-      )?.value !== "false";
+    if (this.useChapters === null) {
+      this.myUseChapters =
```

Comment:
Ah, I  now understand. It is the mix between the prop and the settings

Reply:
yes , the my.. is old school java naming convention ( js used to use _foo ) to my knowledge js/vue does not have convention  for this as injection the data via prop in way data is passed in most examples i see